ankihub/main/reset_local_changes.py

```diff
@@ -46,6 +46,8 @@ def reset_local_changes_to_notes(
         subdecks_for_new_notes_only=deck_config.subdecks_enabled,
         suspend_new_cards_of_new_notes=deck_config.suspend_new_cards_of_new_notes,
         suspend_new_cards_of_existing_notes=deck_config.suspend_new_cards_of_existing_notes,
+        # TODO Warn user if full sync is required when resetting local changes
+        raise_if_full_sync_required=False,
```

Comment:
Do you plan to tackle this in this PR?

Reply:
No, I think this warning is not that important. I'll look into it and perhaps add this in a different PR